.github/workflows/tests.yml

```diff
@@ -0,0 +1,146 @@
+name: CI/CD Pipeline
+on: [pull_request]
+
+jobs:
+  rubocop:
+    name: Rubocop
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Prepare Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.4.6
+          bundler-cache: true
+      - name: Rubocop report
+        env:
+          FORCE_COLOR: 1
+        run: bundle exec rubocop --color --fail-fast
+
+  unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.4.6
+        bundler-cache: true
+    - uses: ./.github/actions/setup-test-environment
+    - name: Get yarn cache path
+      id: yarn-cache-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
```

Comment:
This might need an `output` setting like in https://github.com/publiclab/plots2/pull/9998/files#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957fR62 ???

Reply:
ah no, actually let's switch to this easier approach: https://github.com/marketplace/actions/yarn-node-install-cache

`- uses: egordm/gha-yarn-node-cache@v1`

Reply:
no, see below